uc8151/uc8151.go

```diff
@@ -287,32 +308,43 @@ func (d *Device) ClearBuffer() {
 
 // Size returns the current size of the display.
 func (d *Device) Size() (w, h int16) {
-	if d.rotation == ROTATION_90 || d.rotation == ROTATION_270 {
+	if d.rotation == drivers.Rotation90 || d.rotation == drivers.Rotation270 {
 		return d.height, d.width
 	}
 	return d.width, d.height
 }
 
+// Rotation returns the currently configured rotation.
+func (d *Device) Rotation() drivers.Rotation {
+	return d.rotation
+}
+
 // SetRotation changes the rotation (clock-wise) of the device
-func (d *Device) SetRotation(rotation Rotation) {
+func (d *Device) SetRotation(rotation drivers.Rotation) error {
 	d.rotation = rotation
+	return nil
+}
+
+// Set the sleep mode for this display. SInce it is e-ink, it is not really sleep mode. :)
+func (d *Device) Sleep(sleepEnabled bool) error {
+	return nil
 }
```

Comment:
There is in fact a deep sleep mode, which seems to reduce current quite a bit compared to being on all the time. It doesn't sleep the display, but it does put the controller chip to sleep (I think).
I think it would be a good idea to either implement this or note this in the comment.

Reply:
I think I have implemented this now. PTAL.